rust/scx_utils/src/topology.rs

```diff
@@ -103,6 +105,14 @@ lazy_static::lazy_static! {
     /// disabled CPUs that may not be onlined, whose IDs are lower than the
     /// IDs of other CPUs that may be onlined.
     pub static ref NR_CPUS_POSSIBLE: usize = libbpf_rs::num_possible_cpus().unwrap();
+
+    /// The range to search for when finding the number of physical cores
+    /// assigned to a partition to split a large number of cores that share
+    /// an LLC domain. The suggested split for the cores isn't a function of
+    /// the underlying hardware's capability, but rather some sane number
+    /// to help determine the number of CPUs that share the same DSQ.
+    pub static ref NR_PARTITION_MIN_CORES: usize = 4;
+    pub static ref NR_PARTITION_MAX_CORES: usize = 8;
```

Comment:
Having default values is fine but I think schedulers / users should be able to supply these as parameters.